Automattic/wp-calypso
# Checkout: Extract footer from Thank You page

This pull request addresses part of #2484 by moving the code for the support section of the `Thank You` page into its own component. This is an attempt at reducing the size of the `<CheckoutThankYou />` component as well as the first step before displaying a different copy and buttons based on the product just purchased.

![screenshot](https://cloud.githubusercontent.com/assets/594356/13362924/4bcac156-dcc7-11e5-8c8c-104a04e94340.png)

#### Testing instructions
 
1. Run `git checkout update/thank-you-support-section` and start your server
2. Purchase any type of product
3. Check that the `Thank You` page displays as usual

You might want to try with a Jetpack site since the support section is slightly different in that case. 
 
#### Reviews
